assets/xml/scenes/overworld/spot06.xml

```diff
@@ -2,8 +2,11 @@
     <File Name="spot06_scene" Segment="2">
         <Cutscene Name="gLakeHyliaFireArrowsCS" Offset="0x7020"/>
         <Cutscene Name="gLakeHyliaOwlCs" Offset="0x1B0C0"/>
-        <Path Name="spot06_scenePathList_007764" Offset="0x7764" NumPaths="2"/>
         <Cutscene Name="gLakeHyliaIntroCs" Offset="0x7A30"/>
+        <Cutscene Name="gLakeHyliaCreditsCs" Offset="0x6E90"/>
+        <Cutscene Name="gLakeHyliaWaterRaisingCs" Offset="0x55E0"/>
+
```

Comment:
The corresponding flag for this cs was just named something to the effect of "lake hylia restored" so maybe we make them consistent. i dont care too much either way

Reply:
oh yeah, its still an open pr https://github.com/zeldaret/oot/pull/2330

Reply:
ah I missed this PR, I'll rename it `gLakeHyliaRestoredCs` then, I think naming it `gLakeHyliaRestoredLakeHyliaCs` is too redundant (or maybe `gLakeHyliaRestoringCs` would fit better since this is during the restoration and not after?)